src/services/platform.services.js

```diff
@@ -0,0 +1,47 @@
+import * as childProcess from 'child_process';
+import * as os from 'os';
+import * as path from 'path';
+import { remote } from 'electron';
+import { PACKAGE_MANAGER } from '../config/app';
+
+// Returns true if the OS is Windows
+export const isWin = (): boolean => /^win/.test(os.platform());
+
+// Returns path to the users Documents direactory
+export const getWindowsHomeDir = (): string => {
+  // For Windows Support
+  // Documents folder is much better place for project folders (Most programs use it as a default save location)
+  // Since there is a chance of being moved or users language might be differet we are reading the value from Registery
+  // There might be a better solution but this seems ok so far
```

Comment:
Nit: couple typos in the comment:

- differet -> different
- Registery -> Registry

Reply:
OK, fixed typos.